scripts/setup-kind-multicluster.sh

```diff
@@ -42,7 +42,7 @@ EOF
 }
 
 registry_name='kind-registry'
-registry_port='5000'
+registry_port='5001'
```

Comment:
Why would it break the setups of others? My understanding is that the airplay service on Mac runs on port 5000 and this is consistent across all MacOS machines. I do not know if you can configure this port but I suspect it might take some finagling around relatively deep in the system.

Is there another commonly used service on port 5000 which others in the team are using and which will block this fix from working? I know we can't guarantee that nothing is on 5000 globally, but if there is a service on that port on the most common OS used in the team, trying to use it for the registry seems like a bad idea.